frontend/src/features/documents/list/DocumentFilter/DocumentFilterForm.tsx

```diff
@@ -0,0 +1,105 @@
+import { ResetButton, SearchButton } from 'components/common/buttons';
+import { Form, Select, SelectOption } from 'components/common/form';
+import * as API from 'constants/API';
+import { Formik } from 'formik';
+import { useApiDocuments } from 'hooks/pims-api/useApiDocuments';
+import useIsMounted from 'hooks/useIsMounted';
+import useLookupCodeHelpers from 'hooks/useLookupCodeHelpers';
+import { defaultDocumentFilter, IDocumentFilter } from 'interfaces/IDocumentResults';
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+
+export interface IDocumentFilterFormProps {
+  documentFilter?: IDocumentFilter;
+  onSetFilter: (filterValues: IDocumentFilter) => void;
+}
+
+export const DocumentFilterForm = (props: IDocumentFilterFormProps) => {
+  const { getOptionsByType } = useLookupCodeHelpers();
+  const documentStatusTypeOptions = getOptionsByType(API.DOCUMENT_STATUS_TYPES);
+  const isMounted = useIsMounted();
+  const { getDocumentTypes } = useApiDocuments();
+  const [documentTypes, setDocumentTypes] = React.useState<SelectOption[]>([]);
+
+  React.useEffect(() => {
+    getDocumentTypes().then(({ data }) => {
+      if (data && isMounted()) {
+        setDocumentTypes(
+          data.map(dt => {
+            return {
+              label: dt.documentType,
+              value: dt.id ? dt.id.toString() : '',
+            };
+          }),
+        );
+      }
+    });
+  }, [isMounted, getDocumentTypes]);
```

Comment:
We can not include the documentType in the lookup response/service because it s not having the DB schema same as other lookup DB tables, like DisplayOrder and Description fields are missing in the Document Type table and when we try to put it in the Lookup it does not allow.

Reply:
good to know. thanks for checking!